spyder/plugins/help/utils/static/dark_css/default.css

```diff
@@ -425,3 +425,53 @@ div.admonition p.admonition-title {
 .panel-danger > .panel-body {    
     background-color: #E74C3C;
 }
+
+/* --- Scrollbar -- */
+
+::-webkit-scrollbar {
```

Comment:
I assume using experimental ``-prefix-`` properties is the only way to get this to work, or is there a way to avoid engine-specific hacks?

Reply:
As far as I know there is no way to style the scrollbar of a QWebEngineView element besides using these properties. From what I searched there is a way to hide the scrollbars using QWebEngineSettings::ShowScrollBars but no more than that.